packages/core/admin/index.js

```diff
@@ -92,17 +92,30 @@ async function build({ plugins, dir, env, options, optimize }) {
 async function createPluginsJs(plugins, dest) {
   const pluginsArray = plugins.map(({ pathToPlugin, name }) => {
     const shortName = _.camelCase(name);
+
+    /**
+     * path.join, on windows, it uses backslashes to resolve path.
+     * The problem is that Webpack does not windows paths
+     * With this tool, we need to rely on "/" and not "\".
+     * This is the reason why '..\\..\\..\\node_modules\\@strapi\\plugin-content-type-builder/strapi-admin.js' was not working.
+     * The regexp at line 105 aims to replace the windows backslashes by standard slash so that webpack can deal with them.
+     * Backslash looks to work only for absolute paths with webpack => https://webpack.js.org/concepts/module-resolution/#absolute-paths
+     */
+    const realPath = path
```

Comment:
can you try `const realPath = path.join(pathToPlugin, 'strapi-admin.js')` and most importantly remove the relative path part ? that would allow webpack resolution to work without doing path manipulations.